### PR TITLE
LRAC-8564, LRAC-8565, LRAC-8566, LRAC-8567, LRAC-8569, LRAC-8570, LRAC-8571 | master

### PR DESCRIPTION
**Pull request by Sharryxue**
https://issues.liferay.com/browse/LRAC-8564
https://issues.liferay.com/browse/LRAC-8565
https://issues.liferay.com/browse/LRAC-8566
https://issues.liferay.com/browse/LRAC-8567
https://issues.liferay.com/browse/LRAC-8569
https://issues.liferay.com/browse/LRAC-8570
https://issues.liferay.com/browse/LRAC-8571

Commits are only for master. Thanks
/cc @alexcandidos @christianstokes

### CODE REVIEW

**Comment by Sharryxue** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
ci:test

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase [hunk 1]
```diff
@@ -75,30 +368,181 @@ definition {
...
 
-		// TODO LRAC-8569 CanAddSegmentWithCustomFieldsForIndividuals pending implementation
+			ACSegments.createDynamicSegment();
 
+			ACSegments.nameSegment(segmentName = "Dynamic Segment Using Individual Property Custom Fields");
+
+			ACSegments.goToSidebarAttributes(criterion = "Individual Attributes");
+
+			ACSegments.addSegmentField(segmentField = "Custom Field");
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase:465]: hi @Tim-Cao , I think we forgot to create the Custom Field for this test. What do you think?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase [hunk 2]
```diff
@@ -75,30 +368,181 @@ definition {
...
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8569 | Test Summary: Add segment with custom field for individuals"
-	@ignore = "true"
 	@priority = "4"
 	test CanAddSegmentWithCustomFieldsForIndividuals {
+		task ("Get site assigned property name") {
+			var propertyName = ACDXPSettings.getAssignedPropertyName(siteName = "Site Name");
+		}
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationIndividualAttributes.testcase:438]: We probably would need to create the custom field on DXP at this point, something like was made on the other test cases:

```
		task ("Add Custom Fields") {
			ApplicationsMenu.gotoPortlet(
				category = "Configuration",
				panel = "Control Panel",
				portlet = "Custom Fields");

			CustomFields.addCP(
				customFieldName = "AC Org Custom Fields",
				customFieldType = "Text Area",
				resourceName = "Document");
		}
```

**Review by alexcandidos — COMMENTED** [review]
Hi @Sharryxue , thanks for working on it!

Hi @Tim-Cao , could you please review the point I highlighted for one of the cases?

Also, according to @Sharryxue , she was not able to get the tests passing through automation, according to her comments here https://issues.liferay.com/browse/LRAC-8564 the assert of the last task was failing. She belives it was due to some network issues or maybe it is also related to timing, since manually she was able to see the data on AC in 1-2 minutes.

Could you please update the cases adding them to quarantine if applicable?

Thanks!

**Comment by alexcandidos** [comment]
Aside from that, the code looks fine to me! :)

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-qa/liferay-portal/pull/767

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**